src/API/test/Configurations/useCreateMutation.test.js

```diff
@@ -0,0 +1,73 @@
+import { server, rest, API_BASE } from '../../../test/net';
+import { renderAPIHook, ERROR_RESPONSE } from '../setup'; // must be imported before the tested hooks
+
+import { useCreateMutation, useListQuery } from '../../Configurations';
+
+
+const data = {
+  name: 'RS1',
+  providerName: 'DEMATIC_EMS',
+};
+
+const url = {
+  create: `${API_BASE}/configurations`,
+  list: `${API_BASE}/configurations`,
+};
+
+let request;
+
+beforeEach(() => {
```

Comment:
outside describe? is it legal?

Reply:
Yes, perfectly so.
Took it from this exapmle https://testing-library.com/docs/react-testing-library/example-intro/

Reply:
IMO describe provides more readable message in case of fail

Reply:
`describe` is just a tool [for grouping related test](https://jestjs.io/docs/api#describename-fn).
Test files are the tool for the same purpose.

If the filename clearly tells what the test is about, the message is informative enough
![image](https://user-images.githubusercontent.com/72964370/120212453-ae1d9c80-c23a-11eb-97fe-3abf1b81a351.png)

Reply:
What is nice about using `describe` is that you can filter the entire test file with `jest -t "some describe text"`. Without it it's harder to do.